cli/src/commands/operation/log.rs

```diff
@@ -60,9 +60,13 @@ pub struct OperationLogArgs {
     /// Don't show the graph, show a flat list of operations
     #[arg(long)]
     no_graph: bool,
-    /// Render each operation using the given template
+    /// Render each operation using the given [template expression] with the
+    /// [built-in keywords]
```

Comment:
I don't have strong feeling about `<URL>` vs `[anchor]`, but I wouldn't add hyperlink marker to the summary line, which is displayed in the short help.